project/ntut-xuan/README.rst

```diff
@@ -0,0 +1,198 @@
+PolygonDust
+===========
+
+Description
+-----------
+
+PolygonDust is an application to measure the area size of polygon.
+
+“Dust” means we trying to shred the image into lot of pieces and measure
+polygon size by these pieces.
+
+.. image:: https://media.discordapp.net/attachments/950048467294760990/1284862080050135071/image.png?ex=66e82c8f&is=66e6db0f&hm=8f52c071f46be06d7d6d014e8551087f495e2cf1332a784079c4805429e7aea3&=&format=webp&quality=lossless&width=2880&height=848
+
+Problem to Solve
+----------------
+
+In size measurement of ploygon, we have two methods can do it.
+
+-  Pixel-Method: Shred the image to lot of rectangles and calculate the
+   size by these rectangles.
+-  Formula-Method: Calculate the polygon by formula.
+   e.g. :math:`W \times H` for rectangle of :math:`r^2 \times \pi` for
+   circle.
+
+PolygonDust is used to solve size calculation by Pixel-Method. In
+polygon description, we use image instead formula, that may make the
+problem more easier to solve.
+
+Expect the process steps should be:
+
+-  `Gaussian
+   Blur <https://zh.wikipedia.org/zh-tw/%E9%AB%98%E6%96%AF%E6%A8%A1%E7%B3%8A>`__
+   to decrease the image noice.
+-  Binarization: We trying to use `Otsu’s
+   method <https://zh.wikipedia.org/wiki/%E5%A4%A7%E6%B4%A5%E7%AE%97%E6%B3%95>`__,
+   it calculate the threadhold globally, and may have more efficiency to
+   process binarization.
+-  Shred the image to lot of piece
+-  Process these piece by determine the piece is fill/not fill, and vote
+   0/1.
+-  By the result of voting, calculate the area of ploygon.
+
+The gaussian blur should implement matrix calculation and Otsu’s methods
+should implement K-means algorithm to solve the problem. We will
+implement numerical process in C/C++ to keep its efficientcy.
+
+Prospective Users
+-----------------
+
+Since Fomula-Method to describe the polygon may have some obstacle on
```

Comment:
Is there a reference for the term "formula method"?  If yes, please let me know.

I have heard of geometry-based or vector-based method, which are described in computer geometry.  It is the first time I hear the term "formula method" and would like to know the reference.

Reply:
Since I don't know how to descirbe that "Use mathematics way to calculate the polygon", I just call it "formula method". I think it may confused the reader.

Reply:
You can take a look at the island format and trap format of polygons.